cmake/libprotobuf-lite.cmake

```diff
@@ -75,6 +75,6 @@ if(MSVC AND protobuf_BUILD_SHARED_LIBS)
 endif()
 set_target_properties(libprotobuf-lite PROPERTIES
     VERSION ${protobuf_VERSION}
-    OUTPUT_NAME ${LIB_PREFIX}protobuf-lite
-    DEBUG_POSTFIX "${protobuf_DEBUG_POSTFIX}")
-add_library(protobuf::libprotobuf-lite ALIAS libprotobuf-lite)
+    OUTPUT_NAME ${LIB_PREFIX}protobuf-lite${ver}
+    DEBUG_POSTFIX "${CMAKE_DEBUG_POSTFIX}")
+add_library(${nameSpace}libprotobuf-lite ALIAS libprotobuf-lite)
```

Comment:
this is an interesting thing protobuf cmake authors are doing... I can't say that I've ever seen the cmake namespace used in an `add_library()` call...